### PR TITLE
topology: Add CML topology support for rt5682+max98357a variant

### PR DESCRIPTION
**Pull request by sathyap-chrome**
Add new PCMs and backends for CML chromebook with codecs rt5682+max98357a.

Signed-off-by: Sathya Prakash M R <sathya.prakash.m.r@intel.com>

### CODE REVIEW

**Comment by ranj063** [comment]
@sathyap-chrome there are conflicts in this PR. we've recently change cmakelists.txt for topology. Can you please align with it?

@lgirdwood im also thinking should we merge this with sof-cml-rt5682?

**Comment by lgirdwood** [comment]
@ranj063 yes, but matybe better it could #include sof-cml-rt5682 and hence it only needs to define max98357 data.

**Comment by ranj063** [comment]
@lgirdwood makes sense. I'll work with @sathyap-chrome tonight to fix this

**Comment by sathyap-chrome** [comment]
Updated as suggested. @lgirdwood  @ranj063 . 
let me know if its fine this time.

File: tools/topology/sof-cml-rt5682-max98357a.m4 [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Cannonlake DSP configuration
+include(`platform/intel/cnl.m4')
+include(`platform/intel/dmic.m4')
```
Comment (lgirdwood) [inline tools/topology/sof-cml-rt5682-max98357a.m4]: Do we need all the above includes ? (since they will already be included by sof-cml-rt5682.m4).

Comment (sathyap-chrome) [reply tools/topology/sof-cml-rt5682-max98357a.m4]: makes sense.. will remove.

**Pull request merged**